### DISCUSSION TITLE
How can i disable sorting of story in component

### DISCUSSION THREAD

**Discussion by LongHP105** [storybookjs/react-native#599] [Q&A]
![Screenshot 2024-09-19 at 00 14 43](https://github.com/user-attachments/assets/cb309753-5258-40b7-8676-9fde13233361)

How can I disable the sorting of these stories, I want to sort follow my order which I define in *.stories.tsx file

<img width="359" alt="Screenshot 2024-09-19 at 00 16 32" src="https://github.com/user-attachments/assets/e2c6c233-b0dc-4e8d-9431-5df0df91c7de">

The default will be the first, and the Class Name will be the last

**Comment by dannyhw** [comment]
I don't think disabling is not possible currently but you can change the order

Can you try something like this in the meta object

```js
  parameters: {
    options: {
      storySort: {
        order: ['Default', 'Disabled', "Outline", /* ... etc ... */ ],
      },
    },
  },
```

https://storybook.js.org/docs/writing-stories/naming-components-and-hierarchy#sorting-stories

You can also try some of the other stuff in the docs but some of this code is coming from storybook core and other parts are rn specific so its not always clear to me which things will or won't work 😅

**Reply by LongHP105** [reply]
Thank you for your answer, but this option is just for the globally scope.
```
 ERROR  The storySort option parameter can only be set globally
```
so i can not use it.

**Reply by dannyhw** [reply]
You should be able to use it on the meta object (default export) just not on an individual story. 

Another less ideal option is to put numbers in the names like 1Default 2Something

**Reply by LongHP105** [reply]
@dannyhw yes, i already did that, but it's not for story in component, it's for story of project.
